sdk/servicebus/azure-servicebus/tests/test_sessions.py

```diff
@@ -47,7 +47,7 @@
 
 
 class ServiceBusSessionTests(AzureMgmtTestCase):
-    @pytest.mark.skip(reason="TODO: iterator support")
+    @pytest.mark.skip(reason="TODO: Pyamqp Message Serialization Error")
```

Comment:
What kind of error was this?

Reply:
`Failed: [undefined]azure.servicebus.exceptions.MessageSizeExceededError: Serialization operation failed due to unsupported type System.Byte[]. `

Reply:
I was thinking this was related to trying to send a message that we received/peeked and the formatting being off slightly

Reply:
That looks like a bug - we shouldn't skip this.
If we don't want to deal with it in this PR, please open an issue for it and put the issue number in the TODO comment.